pkg/mobile/mobileClient.go

```diff
@@ -66,3 +73,49 @@ func (r *MobileClientRepoImpl) DeleteByName(name string) error {
 func (r *MobileClientRepoImpl) Create(app *v1alpha1.MobileClient) error {
 	return sdk.Create(app)
 }
+
+type MobileWatcher struct {
+	events chan watch.Event
+}
+
+func (w MobileWatcher) Stop() {
+	close(w.events)
+	index := 0
+	for ; index < len(handler.watchers); index++ {
+		if handler.watchers[index] == w {
+			break
+		}
+	}
+	handler.watchers = append(handler.watchers[:index], handler.watchers[index+1:]...)
```

Comment:
Probably better to move the few lines here to become the `removeWatcher` method of the MobileHandler